### DISCUSSION TITLE
post/terraform-azurerm-caf-enterprise-scale-core/

### DISCUSSION THREAD

**Discussion by giscus** [diegosrp/diegosrp.github.io#13] [General]
# post/terraform-azurerm-caf-enterprise-scale-core/

Diego’s personal blog website.

https://diegosrp.github.io/post/terraform-azurerm-caf-enterprise-scale-core/

**Comment by guimachadooo** [comment]
such nice content! that's what I was looking for to my terraform project thank you!
